src/ui/utils.js

```diff
@@ -241,7 +241,8 @@ export function fillToolbar( { view, icons, toolbar, labels, propertyName, nameT
 
 		button.set( {
 			label: labels[ name ],
-			icon: icons[ name ]
+			icon: icons[ name ],
+			tooltip: labels[ name ]
```

Comment:
@panr Those changes needs tests also ;) Sorry for not marking that earlier.

Reply:
Oh, sorry. Classic me. It's not your fault 😎

Reply:
@jodator added ;-)